api/src/controllers/utils.ts

```diff
@@ -15,7 +15,7 @@ export function getNextOpeningDates(
   // 35 is somewhat arbitrary,
   // we just need to have at least 12 once all the closed days and/or delivery time have been removed
   const dateNow = new Date();
-  const dateList = [...Array(35).keys()].map(day => addDays(dateNow, day));
```

Comment:
Do we need to update the comment too?

Reply:
Updated, along with the one further below about formatting the exceptional closed days